### ISSUE TITLE
module 'hs._asm.axuielement' not found

### ISSUE THREAD

**Issue by Jks15063** [ashfinal/awesome-hammerspoon#14]
Hey, installed as instructed by the readme.  I'm getting an [error](https://gist.github.com/Jks15063/a255c32777281bca30226df093227828#file-scratch-lua-L64) 

I'm on os x Sierra, it seems this may be a known issue: https://github.com/asmagill/hammerspoon_asm/issues/3

**Comment by ashfinal** [comment]
I noticed you may install a very early version. Please use `cd ~/.hammerspoon && git fetch` to install the lastest version, and try again.

**Comment by Jks15063** [comment]
hey ashfinal, thanks for the reply.  I double checked that hammerspoon and this project were both up to date, but I still get the same error.  I can't seem to find much on that module other than the link in the OP.  I guess I'll  keep my eye on it.

**Comment by ashfinal** [comment]
That's strange. I found in your error log there are some hotkeys which have been dropped and should not appear. Have you tried emptying your configuration folder before doing anything else? 
I found this work for me:
First, `rm -rf ~/.hammerspoon`
then, `git clone --depth 1 https://github.com/ashfinal/awesome-hammerspoon.git ~/.hammerspoon`

By the way, my hammerspoon version is stable release 0.9.52.
